components/blog/ImageSlider.tsx

```diff
@@ -0,0 +1,42 @@
+"use client";
+import { Fade } from "react-slideshow-image";
+import Image from "next/image";
+import { getStrapiMedia } from "../../utils/blog/api-helpers";
+
+interface Image {
+  id: number;
+  attributes: {
+	alternativeText: string | null;
+	caption: string | null;
+	url: string;
+  };
+}
+
+interface SlidShowProps {
+  files: {
+	data: Image[];
+  };
+}
+
+const Slideshow = ({ data }: { data: SlidShowProps }) => {
+  return (
+	<div className="slide-container">
+	  <Fade>
+		{data.files.data.map((fadeImage: Image) => {
+		  const imageUrl = getStrapiMedia(fadeImage.attributes.url);
+		  if (imageUrl === null) {
```

Comment:
Is there any specific reason why you have only compared it with `null`?

Do we want to handle it differently if it were `undefined`?
```suggestion
		  if (!imageUrl) {
```